<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>425</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Data Editor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/gear.png</normaloff>../icons/gear.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="20" column="8">
    <widget class="QToolButton" name="saveButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../icons/save.png</normaloff>../icons/save.png</iconset>
     </property>
    </widget>
   </item>
   <item row="22" column="3">
    <widget class="QLineEdit" name="columnCreationText">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="22" column="4">
    <widget class="QToolButton" name="columnCreationButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../icons/go.png</normaloff>../icons/go.png</iconset>
     </property>
    </widget>
   </item>
   <item row="20" column="6">
    <widget class="QCheckBox" name="overwriteBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Overwrite</string>
     </property>
    </widget>
   </item>
   <item row="21" column="2">
    <widget class="QToolButton" name="toolButton2">
     <property name="cursor">
      <cursorShape>WhatsThisCursor</cursorShape>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../icons/help.png</normaloff>../icons/help.png</iconset>
     </property>
    </widget>
   </item>
   <item row="20" column="4">
    <widget class="QToolButton" name="filterButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../icons/go.png</normaloff>../icons/go.png</iconset>
     </property>
    </widget>
   </item>
   <item row="20" column="2">
    <widget class="QToolButton" name="toolButton1">
     <property name="cursor">
      <cursorShape>WhatsThisCursor</cursorShape>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../icons/help.png</normaloff>../icons/help.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="19" colspan="11">
    <widget class="QTableWidget" name="tableWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="22" column="2">
    <widget class="QToolButton" name="toolButton3">
     <property name="cursor">
      <cursorShape>WhatsThisCursor</cursorShape>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../icons/help.png</normaloff>../icons/help.png</iconset>
     </property>
    </widget>
   </item>
   <item row="21" column="3">
    <widget class="QLineEdit" name="columnEditText">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="20" column="11">
    <widget class="QToolButton" name="openButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../icons/new folder.png</normaloff>../icons/new folder.png</iconset>
     </property>
    </widget>
   </item>
   <item row="20" column="7">
    <widget class="QToolButton" name="reloadDataButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../icons/reset.png</normaloff>../icons/reset.png</iconset>
     </property>
    </widget>
   </item>
   <item row="22" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Arithmetic operations:</string>
     </property>
    </widget>
   </item>
   <item row="20" column="3">
    <widget class="QLineEdit" name="filterText">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="21" column="4">
    <widget class="QToolButton" name="columnEditButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../icons/go.png</normaloff>../icons/go.png</iconset>
     </property>
    </widget>
   </item>
   <item row="20" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Filter column:</string>
     </property>
    </widget>
   </item>
   <item row="20" column="10">
    <widget class="QLineEdit" name="sepLine">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>,</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="21" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Delete/copy column(s):</string>
     </property>
    </widget>
   </item>
   <item row="20" column="9">
    <widget class="QToolButton" name="commandBoxButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../icons/console.png</normaloff>../icons/console.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
